services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_states.py

```diff
@@ -57,6 +57,8 @@
 
 def extract_task_state(task_status: Dict[str, str]) -> Tuple[ServiceState, str]:
```

Comment:
It would be desirable that the ``_TASK_STATE ...`` map could resolve this situation as well . This "exception to the rule" feels that it breaks the design. It might lead to exception of the exception :-)  

Cleaning up a bit the function shows clearly what I mean:
```python
def extract_task_state(task_status: Dict[str, str]) -> Tuple[ServiceState, str]:

    task_state: ServiceState = _TASK_STATE_TO_SERVICE_STATE[task_status["State"]] 
    last_task_error_msg: str = task_status.get("Err", "")

    if last_errror_msg:
        # THIS IS AN EXCEPTION TO THE GENERAL RULE!!
        # DANGEROUS SINCE IT HIDES ANY POSSIBLE FAILING STATES !!
        return (ServiceState.FAILED, last_errror_msg)
        
    return (task_state, last_task_error_msg)
```

Reply:
Thins will get scrapped, and the there will no longer be any exception, will make the frontend print a message in the logs.